### PR TITLE
Issue 116 plot concentration

### LINKED ISSUE #116: plot concentration in simulate app

### PR DESCRIPTION
**Pull request by martinjrobins**
Fixes #116 

- plots concentrations instead of amounts in simluation app
- adds axis labels to simulation app

### CODE REVIEW

File: pkpdapp/pkpdapp/templates/index.html [hunk 1]
```diff
@@ -6,16 +6,44 @@
...
-		</div>
-	</div>
+<div class="container">
+<h1>
+  Welcome to the PKPDApp, a web-based application to model the
+	distribution and effects of drugs.
+</h1>
+{% if user.is_authenticated %}
+<p>
+Each user has a number of projects that they have created or are assigned to. Use the drop-down in the upper right corner to select from the list of available
```
Comment (ben18785) [inline pkpdapp/pkpdapp/templates/index.html:19]: Not sure this is right now since the drop-down is no longer in the upper right?

**Review by ben18785 — COMMENTED** [review]
Thanks @martinjrobins -- I like the new home page! Just one inline comment about this below.

One thing that doesn't seem to work for me is the PD model simulation: it only changes the plot if I select a minority of biomarkers (tumour volume or body weight). For the rest, the plot remains the same as I select them from the drop down. Any ideas?

In the PD model simulation, the axis label always appears to be just "Biomarker": is this as intended or was this supposed to change when different biomarkers were selected (just wondering because you'd said [here](https://github.com/pkpdapp-team/pkpdapp/pull/119) that the axis labels were added to the simulate app?

File: pkpdapp/pkpdapp/templates/index.html (hunk 1 above)
Comment (martinjrobins) [reply pkpdapp/pkpdapp/templates/index.html:19]: fixed

**Comment by martinjrobins** [comment]
> Thanks @martinjrobins -- I like the new home page! Just one inline comment about this below.
> 
> One thing that doesn't seem to work for me is the PD model simulation: it only changes the plot if I select a minority of biomarkers (tumour volume or body weight). For the rest, the plot remains the same as I select them from the drop down. Any ideas?

That was a bug, this pulldown shouldn't do anything at all (its just for the data). but I've removed this biomarker pulldown alltogether in #121, so its fixed there :)

> 
> In the PD model simulation, the axis label always appears to be just "Biomarker": is this as intended or was this supposed to change when different biomarkers were selected (just wondering because you'd said [here](https://github.com/pkpdapp-team/pkpdapp/pull/119) that the axis labels were added to the simulate app?

Yup, intended just to be "biomarker", the simulatino app shows all the outputs of the model, so could be anything

**Review by ben18785 — APPROVED** [review]
Happy for this to go in since the plotting issue will be addressed in #121

**Pull request merged**